ignite/internal/analytics/analytics.go

```diff
@@ -52,23 +56,46 @@ func SendMetric(wg *sync.WaitGroup, cmd *cobra.Command) {
 		return
 	}
 
-	path := cmd.CommandPath()
-	met := gacli.Metric{
-		Name:      cmd.Name(),
-		Cmd:       path,
-		Tag:       strings.ReplaceAll(path, " ", "+"),
-		OS:        runtime.GOOS,
-		Arch:      runtime.GOARCH,
-		SessionID: dntInfo.Name,
-		Version:   version.Version,
-		IsGitPod:  gitpod.IsOnGitpod(),
-		IsCI:      getIsCI(),
+	versionInfo, err := version.GetInfo(context.Background())
+	if err != nil {
+		return
+	}
+
+	var (
+		path         = cmd.CommandPath()
+		scaffoldType = ""
+	)
+	if strings.Contains(path, "ignite scaffold") {
```

Comment:
Why do we only store ignite scaffold commands? We don't want to record ignite network, app or anything else usage?

Reply:
We also store all commands by level. But Nathalie asked that we also fill in the scaffold type parameter so we can measure with scaffold type are more used. Some data in analytics is a little bit redundant, but it is better to generate the dashboard. like the `action_name`